framework/src/Volo.Abp.AspNetCore/Volo/Abp/AspNetCore/ExceptionHandling/AbpExceptionHandlingOptions.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Volo.Abp.AspNetCore.ExceptionHandling
+{
+    public class AbpExceptionHandlingOptions
+    {
+        public bool SendAllExceptionsToClients { get; set; } = false;
```

Comment:
`SendExceptionsDetailsToClients` would be a better name 😄

Reply:
I think so. 👍